src/java/org/apache/cassandra/tools/nodetool/DataPaths.java

```diff
@@ -38,16 +45,46 @@ public class DataPaths extends NodeToolCmd
     @Option(title = "format", name = {"-F", "--format"}, description = "Output format (json, yaml)")
     private String outputFormat = "";
 
+    @Option(title = "",
```

Comment:
What is the empty title good for? What about "orphaned"?

Reply:
I would also add short form like "-o".

Reply:
I've reverted that commit, as Brandon confirmed my concerns.